setup.py

```diff
@@ -56,13 +56,21 @@
         'Operating System :: POSIX :: Linux',
     ],
 
-    packages=['bigchaindb', 'bigchaindb.commands', 'bigchaindb.db'],
+    packages=[
+        'bigchaindb',
+        'bigchaindb.commands',
+        'bigchaindb.db',
+        'bigchaindb.consensus'
+    ],
 
     entry_points={
         'console_scripts': [
             'bigchaindb=bigchaindb.commands.bigchain:main',
             'bigchaindb-benchmark=bigchaindb.commands.bigchain_benchmark:main'
         ],
+        'bigchaindb.plugins': [
+            'base=bigchaindb.consensus.base:ConsensusRules'
```

Comment:
The way the change is structured, the name here `base=` needs to be different (and should be descriptive) for each plugin. `bigchaindb.consensus` is the namespace.